python/private/common.bzl

```diff
@@ -30,6 +30,15 @@ PackageSpecificationInfo = getattr(py_internal, "PackageSpecificationInfo", None
 # Extensions without the dot
 _PYTHON_SOURCE_EXTENSIONS = ["py"]
 
+# Extensions that make a file considered importable
+PYTHON_FILE_EXTENSIONS = [
+    "py",
+    "so",  # Python C modules, usually Linux
+    "dylib",  # Python C modules, Mac specific
+    "pyc",
```

Comment:
Maybe adding pyi files here would be useful? If I understand correctly this is to decide which folders to symlink?

Reply:
Done.